frontend/src/metabase/dev-noop.js

```diff
@@ -0,0 +1,2 @@
+// Make sure this gets recognized as an ES module
+export const noop = () => {};
```

Comment:
I think `export {}` might work. And probably be better than just exporting something we never used. Could you try that out?

Reply:
You could also use this file which is already used for EE plugins https://github.com/metabase/metabase/blob/4953ab8575a9acbdfe999349e0ecc39e38700663/frontend/src/metabase/lib/noop.js#L5-L4

Reply:
@ranquild using `metabase/lib/noop.js` results in the error:

```
[js] ERROR in ./app-main.js 5:0-22
[js] Module not found: Error: Can't resolve 'metabase-dev' in '/home/kiselev/git/metabase/metabase/frontend/src/metabase'
```

@WiNloSt nice catch! updated